test/check-application

```diff
@@ -474,21 +472,54 @@ class TestApplication(testlib.MachineCase):
         self.assertIn('rw', rwmnt)
         self.assertIn(rwdir[4:], rwmnt)
 
+        if m.image != "fedora-29":
+            b.wait(lambda: "Hello World" in m.execute("podman logs busybox-with-tty"))
+            b.click("a:contains('Console')")
```

Comment:
Could this also be done on F-29, and check that there it shows the "not supported" message properly?

Reply:
Done